fk/secretsend.go

```diff
@@ -77,9 +77,25 @@ https://download.fluidkeys.com#` + recipientEmail + `
 	var secret string
 	var basename string
 	if filename != "" {
-		secret, err = getSecretFromFile(filename, nil, nil)
+		secret, err = getSecretFromFile(filename, nil)
+
+		out.Print("---\n")
+		out.Print(secret)
+		out.Print("---\n\n")
+
+		out.Print(colour.Info("   It will be end-to-end encrypted so no-one else can read it\n\n"))
+
+		prompter := interactiveYesNoPrompter{}
+
+		if !prompter.promptYesNo("Send "+filename+"?", "y", nil) {
```

Comment:
Cool, to confirm, it now doesn't print an error message, it just quits. That seems good to me.

Reply:
That's right.